static/_redirects

```diff
@@ -235,6 +235,7 @@
 /docs/tasks/administer-cluster/default-cpu-request-limit/     /docs/tasks/configure-pod-container/assign-cpu-resource/#specify-a-cpu-request-and-a-cpu-limit/ 301
 /docs/tasks/administer-cluster/default-memory-request-limit/     /docs/tasks/configure-pod-container/assign-memory-resource/#specify-a-memory-request-and-a-memory-limit/ 301
 /docs/tasks/administer-cluster/developing-cloud-controller-manager.md     /docs/tasks/administer-cluster/developing-cloud-controller-manager/ 301
+/docs/tasks/administer-cluster/enabling-endpointslices.md /docs/concepts/services-networking/endpoint-slices.md 301
```

Comment:
This should be:
/docs/tasks/administer-cluster/enabling-endpointslices/  /docs/concepts/services-networking/endpoint-slices/  301